apps/ecosystem/src/ecosystem.service.ts

```
@@ -876,4 +899,33 @@ export class EcosystemService {
   async getEcosystemEnableStatus(): Promise<boolean> {
     return this.ecosystemRepository.getEcosystemEnableStatus();
   }
+
+  async getEcosystemOrgs(
+    orgId: string,
+    pageDetail: IPaginationSortingDto
+  ): Promise<PaginatedResponse<IGetEcosystemOrgsResponse>> {
+    const rawData = await this.ecosystemRepository.getEcosystemOrgs(orgId, pageDetail);
+    const data = rawData.data.map((item) => {
+      const [leadData] = item?.ecosystem?.ecosystemOrgs;
```

Biome (2.4.4)

[error] Unsafe usage of optional chaining.

(lint/correctness/noUnsafeOptionalChaining)